packages/experimental/Button/src/CompoundButton/CompoundButton.tsx

```diff
@@ -47,4 +47,8 @@ export const CompoundButton = compose<CompoundButtonType>({
   },
 });
 
+export const CompoundButton = React.forwardRef<IFocusable, CompoundButtonProps>((props, ref) => (
+  <CompoundButtonComposed {...props} innerRef={ref} />
+));
```

Comment:
oh, the name change because we're doing this

Reply:
Would this work?
```js
const Button = React.forwardRef<IFocusable, CompoundButtonProps>((props, ref) => (
  compose<CompoundButtonType>({...})
)
```

compose<CompoundButtonType>({

Reply:
I've tried this but I haven't gotten it to work, unfortunately. The return type of compose doesn't match what forwardRef expects. It's also not clear how to pass the props/ref args into compose...

Reply:
Ah rip :/